chrome/content/coniks/context.js

```diff
@@ -0,0 +1,41 @@
+this.EXPORTED_SYMBOLS = ["AccountContext", "ConiksContext"];
+
+var { interfaces: Ci, utils: Cu, classes: Cc } = Components;
+
+Cu.import("chrome://otr/content/helpers.js");
+
+class ConiksContext {
+  constructor(opts) {
+    opts = opts || {};
```

Comment:
Ya, there's gotta be a way to use destructuring to just get the fields you want.